changelog.md

```diff
@@ -1,4 +1,32 @@
 
+## 3.0.0 / 2020-08-XX
+
+### New Features
+
+- Tracim Live Message is now available. Its now not necessary to refresh Tracim web page manually, all new contents information are automatically available in your web browser
+- Tracim permit connection with a username instead of email. This username feature is mandatory to introduce mention feature (available in next release)
```

Comment:
```suggestion
- Tracim now automatically and instantly refreshes its interface when documents and comments are created, modified or deleted. It is not necessary to reload Tracim's web page anymore to see what changed.
- It is now possible to log in using a username instead of one's email address. Usernames will be used for mentions, a feature that will be available in the next release.
```

Should we mention that email addresses become optional?

Reply:
Good question. By default I'd say no, but its not really easy to know that.Good question. By default I say no, but its not really easy to know that.

Can you purpose something to explain that ?

Reply:
Ideo from @raphj 
~~~
Tracim can now be configured to allow user account creation without an email address
~~~

I have added this in changelog.